src/models/roleModel.js

```diff
@@ -0,0 +1,18 @@
+import {BaseModel} from 'nxus-storage'
+
+export default BaseModel.extend({
```

Comment:
Didn't we have a method of using the native `extend` for these base model classes?